backend/routes/postReaction.py

```
@@ -25,207 +25,180 @@ def get_db():
+    return {
+        "id": comment.id,
+        "content": comment.content,
+        "created_at": comment.created_at,
+        "user": serialize_user(comment.user),
+        "total_likes": len(comment.likes),
+        "user_liked": any(l.user_id == user.id for l in comment.likes),
+        "replies": [build_reply_response(r, user) for r in replies]
+    }
+
 
+def build_reply_response(reply, user: User):
```

Ruff (0.8.2)

Missing return type annotation for public function `build_reply_response`

(ANN201)